web/src/routes/SearchPage.tsx

```diff
@@ -25,6 +25,13 @@ const List = styled.ul`
   }
 `
 
+const SearchCounter = styled.p`
+  margin: 10px 0;
+  padding-left: 5px;
+  fontsize: 'small';
```

Comment:
```suggestion
  font-size: 'small';
```

Also "small" is not valid value, but you can reuse the size defined in the theme
`font-size: ${props => props.theme.fonts.decorativeFontSizeSmall};`